src/views/Update/UpdatePage/index.js

```diff
@@ -0,0 +1,257 @@
+        [{value: "Home Phone"}, {value: "Work Phone"}, {value: "Mobile Phone"}]
+      }
+    ];
+    const mailingInfo = [
+      {label: "Street", name: "address", value: userInfo.address},
+      {label: "City", name: "city", value: userInfo.city},
+      {label: "State", name: "state", value: userInfo.state},
+      {label: "Zip", name: "zip", value: userInfo.zip},
+      {label: "Country", name: "country", value: userInfo.country}
+    ];
+    const shouldContactForm = [
+      {label: "Do Not Contact", name: "doNotContact", checked: userInfo.doNotContact},
+      {label: "Do Not Mail", name: "doNotMail", checked: userInfo.doNotMail}
+    ];
+
+    const infoMap = (data) => {
+      return data.map((info) => {
+        let content =
+          typeof info.checked === "boolean"
+          ? <UpdateForm
+              info={info}
+              onChange={handleChange}
+            />
+          : typeof info.menuItem === "object"
+            ? <UpdateSelect
+                info={info}
+                onChange={handleChange}
+              />
+            : <UpdateGrid
+                info={info}
+                onChange={handleChange}
+              />
+        return content;
+      })
+    }
+
+    if (isOnline && isUserStudent) {
+      return (
+        <>
+          <Grid container justifyContent="center">
+            <Grid item xs={12} lg={8}>
+              <Card className={styles.update}>
+                <CardHeader
+                  className={styles.update_title}
+                  title="Update Information"
+                  titleTypographyProps={{ variant: 'h4' }}
+                />
+                <CardContent>
+                  {contentCard({
+                    title: "Personal Information",
+                    info: infoMap(personalInfo)
+                  })}
+                  {contentCard({
+                    title: "Email Addresses",
+                    info: infoMap(emailInfo)
+                  })}
+                  {contentCard({
+                    title: "Phone Numbers",
+                    info: infoMap(phoneInfo)
+                  })}
+                  {contentCard({
+                    title: "Mailing Address",
+                    info: infoMap(mailingInfo)
+                  })}
+                  {contentCard({
+                    title: "Contact Preferences",
+                    info: infoMap(shouldContactForm)
+                  })}
+                  <Grid item xs={12} justifyContent="center">
+                    {saveButton}
+                  </Grid>
+                </CardContent>
+              </Card>
+              <Typography variant="subtitle1">
+                Found a bug?
+                <a href="mailto:cts@gordon.edu?Subject=Gordon 360 Bug">
+                  <Button
+                    style={{ color: gordonColors.primary.cyan }}>Report to CTS</Button>
+                </a>
```

Comment:
Rather than wrapping the button in an anchor tag, you can just set the `href` prop of the button and it will behave like a link.

Note though that you should only do this for non-navigational links like `mailto`. `http[s]` links should use the `Link` component from react router for proper navigation).

Also, you can just use `color="primary"` to set the color.

```suggestion
                <Button
                    href="mailto:cts@gordon.edu?Subject=Gordon 360 Bug" color="primary">Report to CTS</Button>
```